### PR TITLE
[Feature] 3x3 image grid with hardcode image links

### PR DESCRIPTION
**Pull request by chloeg12**

### CODE REVIEW

**Review by rebeccali04 — COMMENTED** [review]
Hi Chloe,

It seems like there are two .DS_Store files in two difference places that stores the same images, are there any significant differences and are both necessary? I’m guessing this is leftover cache data so not a huge problem.

Rebecca

**Review by arshinx — APPROVED** [review]
lgtm!

File: pages/images.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    alt: "Description of image 9",
+  },
+];
+
+function ImageCell({ src, alt }) {
+    return <img className={styles.image_cell} src={src} alt={alt} />;
+}
+
+export default function ImageGrid() {
+    return (
```
[the remark this answers is not in the document]
Comment (chloeg12) [reply pages/images.js:47]: Do you mean control the total number of images in image list? I am not sure

[the remark this answers is not in the document]
Comment (arshinx) [reply pages/images.js:47]: @chloeg12 nah i'm thinking ImageGrid takes in a parameter: list of images and renders them in a grid

edge case: images don't fit in a square, so just use enough so it fits. ex: input: 11 images, use first 9 of them. if not enough, just show an error message (ex: Error loading images)

**Review by rebeccali04 — APPROVED** [review]
LGTM!

File: pages/images.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (chloeg12) [reply pages/images.js:47]: ok I will fix it

**Review by arshinx — APPROVED** [review]

**Review by rebeccali04 — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**